### DISCUSSION TITLE
[Question]: Please educate me on the topic to use: reviews vs events

### DISCUSSION THREAD

**Discussion by leonbrag** [blakeblackshear/frigate#13362] [Ask A Question] [answered]
### Describe the problem you are having

I am trying to understand what MQTT topic should be used: **reviews or events**

The objective is to send notification on alerts (say ppl, cars), or just general motion (say objects cannot be identified, but motion is detected - example animal is walking by)

### Why I am confused.

I see all the example (like this https://github.com/blakeblackshear/frigate/discussions/2898) is using events topic.
This page says to use reviews, while code sample on page uses events: https://github.com/blakeblackshear/frigate/discussions/2898

It seems events are raised for true positive, should events then be used?

If reviews should be used, what type should I be looking at: new or end?

Note:
I will not be using HASS (not sure it it's relevant) I will use Node Red or will write entire MQTT management backend prob in Python and will use Flask as my UI for it

### Version

.14

### Frigate config file

```yaml
na
```

### Relevant Frigate log output

```shell
na
```

### Relevant go2rtc log output

```shell
na
```

### FFprobe output from your camera

```shell
na
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

OpenVino

### Network connection

Wired

### Camera make and model

na

**Comment by NickM-27** [comment, marked as answer]
events are individual objects that are detected and the messages sent are very verbose so it is really not good for notifications unless you want to do a lot of filtering, reviews match the UI behavior and are only updated when something significant has changed and it also groups activity of all objects together so a person getting out of a car that just arrived would be one notification vs 2 for example so these are recommended for notifications.
